src/bdscript/dm.md

```diff
@@ -22,5 +22,9 @@ $dm[UserID;...]
 ```
 This will send the message to every user with given User ID.
 
+## Parameters 
+- `userID` `(Type : Snowflake || Flag : Emptiable)`

```

Comment:
I'm going to let this pass, since the `$dm[]` function is a bit weird, so I'm not requiring you to split this specific page.
```suggestion
- `userID` `(Type: Snowflake || Flag: Emptiable)`: The user to whom to send the direct message. 

```